tests/x11/yast2_lan_restart.pm

```diff
@@ -87,6 +87,7 @@ sub run {
 sub post_fail_hook {
     assert_script_run 'journalctl -b > /tmp/journal', 90;
     upload_logs '/tmp/journal';
+    $self->SUPER::post_fail_hook;
```

Comment:
Please add `my ($self) = @_;` and we are good to go =)

Reply:
already on it :)